### PR TITLE
Issue373_HPsimpledocumentation

### PR DESCRIPTION
**Pull request by aku-cbe**
Issue #373 states that in HeatPumpSimple there are missing documentations.

In this branch all the missing documentation is added.

Sorry Marcus, I don't know who else to assign.

### CODE REVIEW

**Comment by marcusfuchs** [comment]
@aku-cbe Thanks a lot for the pull request. From this we also learned that in the tutorial, we forgot to mention that this pull request should not go to `master`, but to `development`. I fixed this in the tutorial and will edit your PR accordingly.

File: AixLib/Fluid/HeatPumps/HeatPumpSimple.mo [hunk 1]
```diff
@@ -7,42 +7,54 @@ model HeatPumpSimple
...
+                    "Water volume of the condenser on the sink side"  annotation(Placement(transformation(extent = {{-10, -10}, {10, 10}}, rotation = 90, origin={70,-30})));
   Fluid.Sensors.TemperatureTwoPort
-                            temperatureSinkOut(redeclare package Medium =
-        Medium, m_flow_nominal=0.01)           annotation(Placement(transformation(extent = {{-10, -10}, {10, 10}}, rotation = 90, origin = {80, 50})));
-  Modelica.Blocks.Interfaces.BooleanInput OnOff annotation(Placement(transformation(extent = {{-20, -20}, {20, 20}}, rotation = 270, origin = {0, 80})));
+                    temperatureSinkOut(redeclare package Medium =
+                    Medium, m_flow_nominal=0.01)
+                    "Temperature sensor at outlet on the sink side"   annotation(Placement(transformation(extent = {{-10, -10}, {10, 10}}, rotation = 90, origin = {80, 50})));
+  Modelica.Blocks.Interfaces.BooleanInput OnOff
+                    "On Off input signal of the heatpump"      annotation(Placement(transformation(extent = {{-20, -20}, {20, 20}}, rotation = 270, origin = {0, 80})));
```
Comment (marcusfuchs) [inline AixLib/Fluid/HeatPumps/HeatPumpSimple.mo]: @aku-cbe Just a small typo: `heatpump` should be `heat pump` instead.

File: AixLib/Fluid/HeatPumps/HeatPumpSimple.mo [hunk 2]
```diff
@@ -7,42 +7,54 @@ model HeatPumpSimple
...
-  Modelica.Blocks.Sources.Constant constZero2(k = 0) annotation(Placement(transformation(extent = {{-26, -28}, {-6, -8}})));
-  Modelica.Blocks.Logical.Switch SwitchPower annotation(Placement(transformation(extent = {{14, 12}, {34, 32}})));
-  Modelica.Blocks.Sources.Constant constZero1(k = 0) annotation(Placement(transformation(extent = {{-26, 4}, {-6, 24}})));
-  Modelica.Blocks.Math.Feedback feedbackHeatFlowEvaporator annotation(Placement(transformation(extent = {{10, -60}, {-10, -40}})));
+  Modelica.Blocks.Tables.CombiTable2D PowerTable(table = tablePower)
+                    "calculates electric power based on temperature in source and sink" annotation(Placement(transformation(extent = {{-52, 20}, {-32, 40}})));
+  Modelica.Blocks.Tables.CombiTable2D HeatFlowCondenserTable(table = tableHeatFlowCondenser)
+                    "Calculates heat flow based on temperature in source and sink" annotation(Placement(transformation(extent = {{-52, -12}, {-32, 8}})));
+  Modelica.Blocks.Logical.Switch SwitchHeatFlowCondenser
+                    "Switch to deactivate heatflow when off" annotation(Placement(transformation(extent = {{14, -20}, {34, 0}})));
```
Comment (marcusfuchs) [inline AixLib/Fluid/HeatPumps/HeatPumpSimple.mo]: `heatflow` should be `heat flow`

**Comment by aku-cbe** [comment]
@marcusfuchs 
Thanks a lot.
Could I have edited the target branch myself as well?

File: AixLib/Fluid/HeatPumps/HeatPumpSimple.mo [hunk 3]
```diff
@@ -7,42 +7,54 @@ model HeatPumpSimple
...
+  Modelica.Blocks.Sources.Constant constZero2(k = 0)
+                    "zero heatflow, when heatpump off" annotation(Placement(transformation(extent = {{-26, -28}, {-6, -8}})));
+  Modelica.Blocks.Logical.Switch SwitchPower
+                    "Switch to deactivate Power when off" annotation(Placement(transformation(extent = {{14, 12}, {34, 32}})));
+  Modelica.Blocks.Sources.Constant constZero1(k = 0)
+                    "zero power, when heatpump off" annotation(Placement(transformation(extent = {{-26, 4}, {-6, 24}})));
+  Modelica.Blocks.Math.Feedback feedbackHeatFlowEvaporator
+                    "calculates evaporator heatflow with total energy balance" annotation(Placement(transformation(extent = {{10, -60}, {-10, -40}})));
```
Comment (marcusfuchs) [inline AixLib/Fluid/HeatPumps/HeatPumpSimple.mo]: Some of the above doc-strings start with small letters (e.g. `"calculates..."`, `"zero..."`). The first letter should be capitalized.

**Comment by mlauster** [comment]
Yes, by clicking on the "edit"-button in the upper right corner

**Comment by aku-cbe** [comment]
Oh thanks.
I tried that but somehow I thought it only edits the name of the request.

**Comment by aku-cbe** [comment]
@marcusfuchs 
I changed the requested issues.

**Comment by marcusfuchs** [comment]
@aku-cbe Perfect, thanks again for the first-time contribution and welcome as active developer to AixLib! It would be fantastic if you could summarize your experience with our tutorial in #372 and tell us whether you think this helps new contributors before starting their own development.

**Pull request merged**